docs/pages/database-access/guides/aws-cassandra-keyspaces.mdx

```diff
@@ -0,0 +1,200 @@
+- A host, e.g., an Amazon EC2 instance, where you will run the Teleport Database Service.
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+
+## Step 1/5. Install and configure Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+### Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Proxy Service:
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:443 \
+  --name=keyspaces \
+  --protocol=cassandra \
+  --aws-account-id=12345678912 \
+  --aws-region=us-east-1 \
+  --labels=env=dev
+```
+
+<Admonition type="note">
+
+The `--auth-server` flag must point to the Teleport cluster's Proxy Service
+endpoint because the Database Service always connects back to the cluster over a
+reverse tunnel.
+
+</Admonition>
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Cloud tenant:
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=mytenant.teleport.sh:443 \
+  --name=keyspaces \
+  --protocol=cassandra \
+  --aws-account-id=12345678912 \
+  --aws-region=us-east-1 \
+  --labels=env=dev
+```
+
+</ScopedBlock>
+
+<Admonition type="tip">
+  You can start the Database Service using a configuration file instead of CLI flags.
+  See the [YAML reference](../reference/configuration.mdx) for details.
+</Admonition>
+
+## Step 2/5. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+
+## Step 3/5. Create an AWS Keyspaces role
+
+Create an AWS IAM Role that will be used as your Keyspaces user.
+Go to the IAM -> Access Management -> [Roles](https://console.aws.amazon.com/iamv2/home#/roles).
+Press Create Role.
+
+![Create Role Step 1](../../../img/database-access/guides/keyspaces/create-role-step1.png)
+AWS provides the `AmazonKeyspacesReadOnlyAccess` and `AmazonKeyspacesFullAccess` IAM policies that can incorporated into roles.
+You can choose the `AmazonKeyspacesReadOnlyAccess` for the AWS Keyspaces read-only access or `AmazonKeyspacesFullAccess` for the AWS Keyspaces full access.
+<Admonition type="tip">
+    The `AmazonKeyspacesReadOnlyAccess` and `AmazonKeyspacesReadOnlyAccess` policies may provide too much or not enough access for your intentions. Validate these meet your expectations if you plan on using them. You can also create your own custom AWS Keyspaces Permissions Policies: [Amazon Keyspaces identity-based policy examples](https://docs.aws.amazon.com/keyspaces/latest/devguide/security_iam_id-based-policy-examples.html).
```

Comment:
```suggestion
The `AmazonKeyspacesReadOnlyAccess` and `AmazonKeyspacesReadOnlyAccess` policies may
provide too much or not enough access for your intentions. 
Validate that these meet your expectations if you plan on using them. 
You can also create your own custom AWS Keyspaces Permissions Policies: [Amazon Keyspaces identity-based policy examples](https://docs.aws.amazon.com/keyspaces/latest/devguide/security_iam_id-based-policy-examples.html).
```
This suggestion adds a "that" after "Validate" and breaks up this paragraph for easier editing later on